initialize repo from template (GDEV-955) (#1)
ghga-de/httpyexpect: README.md

![tests](https://github.com/ghga-de/httpyexpect/actions/workflows/unit_and_int_tests.yaml/badge.svg)
[![PyPI version shields.io](https://img.shields.io/pypi/v/httpyexpect.svg)](https://pypi.python.org/pypi/httpyexpect/)
[![PyPI pyversions](https://img.shields.io/pypi/pyversions/httpyexpect.svg)](https://pypi.python.org/pypi/httpyexpect/)
[![Coverage Status](https://coveralls.io/repos/github/ghga-de/httpyexpect/badge.svg?branch=main)](https://coveralls.io/github/ghga-de/httpyexpect?branch=main)

# httpyexpect

> An opinon on how to translate HTTP errors from server to the client side and
> the tools for the implement of that opinion.

Reviewer: ```suggestion
An opinionated way to translate server side HTTP errors to the client side.
```
Something in this direction would be more concise and still fit the intention

Author: Thanks, yes that is much better.
